lib/shared/log/to-console.js

```diff
@@ -37,12 +40,33 @@ function toConsole(log, opts, translate) {
   }
 
   return function () {
+    log.removeListener('deprecated', onDeprecated);
     log.removeListener('error', onError);
     log.removeListener('warn', onWarn);
     log.removeListener('info', onInfo);
     log.removeListener('debug', onDebug);
   };
 
+  function onDeprecated() {
+    if (!deprecatedPrinted) {
+      var msg = { tag: Symbol.for('GULP_CLI_GULPLOG_DEPRECATED') };
```

Comment:
By allowing this to pass through the translation mechanism, users that don't want to see it can return `false` to silence the warning :tada: